### PR TITLE
Add track data for the preview site btn

### LINKED ISSUE #6580: Tracks: Add tracking to Preview site link
In #6513 @moon0326 added the Preview site link in the top bar on the Personalize your store step. Now we'd like to add tracking to this link to measure its success.

Here are the details:

**Event name:** 
wcadmin_tasklist_previewsite

**Event description:** 
When the user clicks on the "Preview site" link from the Personalize your store task list screen

**Event props:**
- create_homepage: Y/N
- upload_logo: Y/N
- set_notice: Y/N

Already chatted with @pmcpinto to confirm these details in Slack.

### PR DESCRIPTION
**Pull request by moon0326**
Fixes #6580 

This PR adds tracking data when a user clicks the `Preview site` button from the activity panel.

### Detailed test instructions:

1. Navigate to WooCommerce -> Home and click "Personalize your store" from the OBW task list or navigate to `/wp-admin/admin.php?page=wc-admin&task=appearance`
2. Open your inspector and enable debugging mode by executing `localStorage.setItem( 'debug', 'wc-admin:*' );` 
3. Click the `Preview site` button and confirm it contains the following event data.

Prop | Possible Values | Note
-----|-----------------|-------
create_homepage| Y or N | when user created homepage from step #2
upload_logo| Y or N | when user uploads a logo from step #3
set_notice | Y or N | when user sets notice text from step #4

### CODE REVIEW

File: TESTING-INSTRUCTIONS.md [hunk 1]
```diff
@@ -2,6 +2,18 @@
 
 ## Unreleased
 
+### Add track data for the preview site btn  #6623
```
Comment (ilyasfoo) [inline TESTING-INSTRUCTIONS.md:5]: I feel testing instruction is unnecessary for this PR since we're not adding any new feature.

**Review by ilyasfoo — CHANGES_REQUESTED** [review]
This tests well for me, though I feel testing instruction is unnecessary

File: TESTING-INSTRUCTIONS.md (hunk 1 above)
Comment (moon0326) [reply TESTING-INSTRUCTIONS.md]: @ilyasfoo that makes sense. Removed in https://github.com/woocommerce/woocommerce-admin/pull/6623/commits/b55c746320a411c07eb456f7da3788aeeacb9572

**Review by ilyasfoo — APPROVED** [review]
LGTM! 🚢

**Pull request merged**